static/js/apps/eval_rig/feedback_form.tsx

```diff
@@ -37,144 +53,202 @@ export interface EvalInfo {
+    <div id="form-container">
+      {evalInfo && (
+        <form onSubmit={handleSubmit}>
+          <fieldset>
+            <div>
+              <h2>OVERALL EVALUATION</h2>
+              <OneQuestion
+                question="How is the overall answer?"
+                name="overall"
+                options={{
+                  LLM_ANSWER_HALLUCINATION: "Found factual inaccuracies",
+                  LLM_ANSWER_OKAY: "No obvious factual inaccuracies",
+                }}
+                handleChange={handleChange}
+                responseField={response.overall}
+                disabled={status === FormStatus.Submitted}
+              />
+            </div>
+
+            <div>
+              <h2>GEMMA MODEL EVALUATION</h2>
+              <h3>{evalInfo.question}</h3>
+              <h3>{evalInfo.llmStat}</h3>
+              <OneQuestion
+                question="Question from the model"
+                name="question"
+                options={{
+                  DC_QUESTION_IRRELEVANT: "Irrelevant, vague, requires editing",
+                  DC_QUESTION_RELEVANT: "Well formulated & relevant",
+                }}
+                handleChange={handleChange}
+                responseField={response.question}
+                disabled={status === FormStatus.Submitted}
+              />
+              <OneQuestion
+                question="Model response quality"
+                name="llmStat"
+                options={{
+                  LLM_STAT_ACCURATE: "Stats seem accurate",
+                  LLM_STAT_INACCURATE: "Stats seem inaccurate",
+                }}
+                handleChange={handleChange}
+                responseField={response.llmStat}
+                disabled={status === FormStatus.Submitted}
+              />
+            </div>
+
+            <div>
+              <h2>DATA COMMONS EVALUATION</h2>
+              <h3>{evalInfo.dcResponse}</h3>
+              <h3>{evalInfo.dcStat}</h3>
+              <OneQuestion
+                question="Response from Data Commons"
+                name="dcResponse"
+                options={dcResponseOptions}
+                handleChange={handleChange}
+                responseField={response.dcResponse}
+                disabled={status === FormStatus.Submitted}
+              />
+
+              <OneQuestion
+                question="Response from Data Commons"
+                name="dcStat"
+                options={{
+                  DC_STAT_ACCURATE: "Stats seem accurate",
+                  DC_STAT_INACCURATE: "Stats seem inaccurate",
+                }}
+                handleChange={handleChange}
+                responseField={response.dcStat}
+                disabled={status === FormStatus.Submitted}
+              />
+            </div>
+          </fieldset>
+        </form>
+      )}
+      <div id="page-screen" className="screen">
+        <div id="spinner"></div>
+      </div>
+    </div>
   );
-}
+});
+
+FeedbackForm.displayName = "FeedbackForm";
```

Comment:
what is this for?

Reply:
not needed as I remove the feedback_form component.